lib/promiscuous_black_hole/operation.rb

```diff
@@ -60,7 +61,7 @@ def destroy
     end
 
     def persist_embedded_records
-      StaleEmbeddingsDestroyer.new(message.table_name, message.id).process
+      StaleEmbeddingsDestroyer.new(message.table_name, message.id).process unless @embedded
```

Comment:
I might be tired, but isn't this backwards? Don't you want to run the `StaleEmbeddingsDestroyer` with embedded stuff?